account_invoice_change_currency/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,5 @@
+Configuration
```

Comment:
What about merge all files in the README file instead?

Or Are there changes in the guidelines that Im missing?
cc @pedrobaeza @sbidoul

Reply:
@pedrobaeza @sbejaoui 

we need your input here.

Regards!

Reply:
Hi @hugho-ad 
I just find the thread about:
 - https://github.com/OCA/maintainer-tools/pull/339

You need to use the current template:
 - https://github.com/OCA/maintainer-tools/tree/master/template/module/readme